source/features/cancel-wiki-edit-button.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'dom-chef';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+async function init(): Promise<void> {
+	(await elementReady('#gollum-editor-submit'))!.after(
+		<a
+			className="flex-auto btn btn-danger"
+			href={location.href.replace(/\/_(edit|new)$/, '')}
```

Comment:
There is another way to do get the previous page: `select('[name="gollum-editor"]').getAttribute('action')`, just FYI.

Reply:
Done